unattended_installer/install_functions/opendistro/wazuh.sh

```diff
@@ -44,17 +44,79 @@ function configureWazuhCluster() {
 function installWazuh() {
 
     logger "Starting the Wazuh manager installation."
+
     if [ "${sys_type}" == "zypper" ]; then
         eval "${sys_type} -n install wazuh-manager=${wazuh_version}-${wazuh_revision} ${debug}"
     else
         eval "${sys_type} install wazuh-manager${sep}${wazuh_version}-${wazuh_revision} -y ${debug}"
     fi
     if [  "$?" != 0  ]; then
         logger -e "Wazuh installation failed"
+        wazuhinstaller="manager"
         rollBack
         exit 1
     else
         wazuhinstalled="1"
         logger "Wazuh manager installation finished."
     fi
 }
+
+
+function uninstallmanager() {
+
+    logger "Wazuh and Filebeat will be uninstalled."
+
+    # remove packages
+    if [[ -n "${wazuhinstalled}" ]];then
+        logger -w "Removing Wazuh packages."
```

Comment:
Why packages? It's only one. 
```suggestion
        logger -w "Removing Wazuh manager."
```

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1211/commits/c3fc83525f9efa17705f53a3ff6e1f8d681d1d49